scripts/splitControl.js

```diff
@@ -58,8 +60,8 @@ var splitControl = function(container, config) {
     let height;
     let width;
 
-    let reSize = $.Callbacks();
-    let dragEnd = $.Callbacks();
+    let reSize = [];
```

Comment:
Alternatively, here and elsewhere, you could replace $.Callbacks with a Set instead of an array. Then the associated add() functions could remain unchanged instead of changing to push(). (It is avaialable within our range of supported browsers.)

Reply:
Perhaps also rename reSize() to onResize() (and prepend on- similarly elsewhwere) to make it clearer what this is doing.